src/helpers/ramlParser.js

```diff
@@ -87,6 +88,14 @@ function _parseRAML(filePath, dest, baseUri, listTraits, cb) {
       trait.is = _avoidRepetitiousTraits(trait.is);
     });
 
+    // extend RAML, dereferencing $refs from JSON schemas
+    try {
+      ramlExtender.extend(data);
+    }
+    catch(err){
+      logger.error(err);
```

Comment:
Maybe add some custom message and then log err, so that we would know what is going on?